docs/fundamentals/executor/executor-in-flow.md

```diff
@@ -146,7 +135,7 @@ with f:
 
 ```yaml
 jtype: MyExecutor
-with:
```

Comment:
we need an example using `with` simply

Reply:
We wanted to keep it consistent across examples, using uses_X in both. In Python you can't use the short form with